src/lib/crypto/kimchi_backend/gadgets/foreign_field.ml

```diff
@@ -317,13 +565,27 @@ end
 
 (* Common auxiliary functions for foreign field gadgets *)
 
+(* TODO: Implement hash function output (e.g. keccak) to ecdsa input integration code *)
+(* Gadget to constrain conversion of bytes array into foreign field element *)
+let _bytes_to_foreign_field_element (type f)
```

Comment:
delete, the code is done in the keccak poc

Reply:
I guess we will delete this whole function once we merge your version of foreign_field.ml from the keccak branch?